entities/project.ts

```diff
@@ -155,6 +155,14 @@ class Project extends BaseEntity {
   @RelationId((project: Project) => project.status)
   statusId: number;
 
+  @Field(type => Float)
+  @Column({ type: 'real', default: 0 })
+  totalDonations: number = 0;
+
+  @Field(type => Float)
+  @Column({ type: 'real', default: 0})
+  totalHearts: number = 0;
```

Comment:
I think we shouldn't name it totalHearts, more like "totalReactions" as we might have more types of reactions in the future and that way it would be more consistent with the user of terms

Reply:
We would change this also in the migration.

Reply:
Yeay it makes sense, 
done